src/Storages/ObjectStorage/DataLakes/DeltaLake/EnginePredicate.cpp

```diff
@@ -386,7 +386,8 @@ uintptr_t EngineIterator::getNextImpl(EngineIteratorData & iterator_data, const
                             /* from */std::move(left_column),
                             /* to */column_node->result_type,
                             DB::CastType::nonAccurate,
-                            std::move(diagnostic));
+                            std::move(diagnostic),
+                            nullptr);
```

Comment:
Not sure if we really need context here, but ideally better to propagate here as well